src/content/operations/troubleshooting/_index.en.md

```diff
@@ -190,15 +190,15 @@ Metadata:
 Status:
   Conditions:
     Last Transition Time:  2020-11-25T06:21:01Z
-    Message:               Awaiting sync of the ServiceImport to the broker
-    Reason:                AwaitingSync
-    Status:                False
+    Message:
```

Comment:
The message is empty, is this intentional?

Reply:
yes - the `Message` field isn't set in this case but it's still outputted,